docs/sources/administration/team-management/index.md

```diff
@@ -19,18 +19,28 @@ For more information about teams, refer to [Teams and permissions]({{< relref ".
 
 For information about how to optimize your teams, refer to [How to best organize your teams and resources in Grafana](https://grafana.com/blog/2022/03/14/how-to-best-organize-your-teams-and-resources-in-grafana/).
 
+This topic describes how to:
+
+- Create a team
+- Add a team member
+- Grant team member permissions
+- Remove a team member
+- Delete a team
+- View a list of teams
+
+## Before you begin
+
+- Ensure that you have either organization administrator permissions or team administrator permissions (Create a team, Grant team member permissions, Remove a team member, View a list of teams)
+- Make a plan for which users belong to which teams and the permissions team members receive (Create a team, Grant team member permissions, Remove a team member, View a list of teams)
+- Ensure that you have organization administrator permissions (Add a team member, Delete a team)
```

Comment:
Do you need to explicitly state which procedures require the permission? This is almost like a matrix of permissions required.  Maybe there is a better way to do this. 

```suggestion
- Ensure that you have either organization administrator permissions or team administrator permissions (Create a team, Grant team member permissions, Remove a team member, View a list of teams)
- Make a plan for which users belong to which teams and the permissions team members receive (Create a team, Grant team member permissions, Remove a team member, View a list of teams)
- Ensure that you have organization administrator permissions (Add a team member, Delete a team)
```